Typescript Examples fail after building

Currently, all the typescript examples will fail after being built. This is because of the `require.resolve` call to `context.ts`. After running the build script (tsc), the compiled files will still reference that `context.ts` but that file will not be there since the outDir for typescript is `dist`.

```
Error: Cannot find module './context.ts'
```

The compiled schema.js will have this line

```
source: require.resolve('./context.ts'),
```

This fails because the file is no longer adjacent to `context.ts` so the relative path breaks.

```bash
# Before - Works Typescript (npm run dev)

src
  schema.ts
  context.ts
```

```bash
# After Compile - Fails Javascript

dist
  schema.js # still references ./context.ts
src
  schema.ts
  context.ts
```

All the **typescript examples** break,  but here's just one, showing where the offending line is:
https://github.com/prisma/prisma-examples/blob/prisma2/typescript/graphql-auth/src/schema.ts#L15


### Steps to reproduce
- `npx prisma2 test`
- Select `Starter kit`
- Select `TypeScript`
- Select `Graphql API`
- Select `SQLite`
- After the project is finished installing, run `npm run build`
- Run the compiled output `node dist/server.js`


### Potential fix
This is what i'm using to make it work

```diff
- source: require.resolve('./context.ts'),
+ source: require.resolve('../src/context.ts'),
```

--- a/misc/react-graphql-fullstack/server/package.json
+++ b/misc/react-graphql-fullstack/server/package.json
@@ -1,6 +1,7 @@
 {
   "name": "typescript-graphql",
   "scripts": {
+    "start": "node dist/server",
     "clean": "rm -rf dist",
     "build": "npm -s run clean && npm -s run generate && tsc",
     "generate": "npm -s run generate:prisma && npm -s run generate:nexus",

--- a/misc/react-graphql-fullstack/server/src/schema.ts
+++ b/misc/react-graphql-fullstack/server/src/schema.ts
@@ -103,7 +103,7 @@ export const schema = makeSchema({
         alias: 'photon',
       },
       {
-        source: require.resolve('./context.ts'),
+        source: require.resolve('./context'),
         alias: 'Context',
       },
     ],

--- a/typescript/graphql-apollo-server/package.json
+++ b/typescript/graphql-apollo-server/package.json
@@ -1,6 +1,7 @@
 {
   "name": "typescript-graphql",
   "scripts": {
+    "start": "node dist/server",
     "clean": "rm -rf dist",
     "build": "npm -s run clean && npm -s run generate && tsc",
     "generate": "npm -s run generate:prisma && npm -s run generate:nexus",

--- a/typescript/graphql-apollo-server/src/schema.ts
+++ b/typescript/graphql-apollo-server/src/schema.ts
@@ -1,8 +1,8 @@
-import { nexusPrismaPlugin } from "nexus-prisma"
-import { idArg, makeSchema, objectType, stringArg } from "nexus"
+import { nexusPrismaPlugin } from 'nexus-prisma'
+import { idArg, makeSchema, objectType, stringArg } from 'nexus'
 
 const User = objectType({
-  name: "User",
+  name: 'User',
   definition(t) {
     t.model.id()
     t.model.name()
@@ -14,7 +14,7 @@ const User = objectType({
 })
 
 const Post = objectType({
-  name: "Post",
+  name: 'Post',
   definition(t) {
     t.model.id()
     t.model.createdAt()
@@ -27,14 +27,14 @@ const Post = objectType({
 })
 
 const Query = objectType({
-  name: "Query",
+  name: 'Query',
   definition(t) {
     t.crud.post({
-      alias: "post",
+      alias: 'post',
     })
 
-    t.list.field("feed", {
-      type: "Post",
+    t.list.field('feed', {
+      type: 'Post',
       resolve: (_, args, ctx) => {
         return ctx.photon.posts.findMany({
           where: { published: true },
@@ -42,8 +42,8 @@ const Query = objectType({
       },
     })
 
-    t.list.field("filterPosts", {
-      type: "Post",
+    t.list.field('filterPosts', {
+      type: 'Post',
       args: {
         searchString: stringArg({ nullable: true }),
       },
@@ -62,13 +62,13 @@ const Query = objectType({
 })
 
 const Mutation = objectType({
-  name: "Mutation",
+  name: 'Mutation',
   definition(t) {
-    t.crud.createOneUser({ alias: "signupUser" })
+    t.crud.createOneUser({ alias: 'signupUser' })
     t.crud.deleteOnePost()
 
-    t.field("createDraft", {
-      type: "Post",
+    t.field('createDraft', {
+      type: 'Post',
       args: {
         title: stringArg({ nullable: false }),
         content: stringArg(),
@@ -88,8 +88,8 @@ const Mutation = objectType({
       },
     })
 
-    t.field("publish", {
-      type: "Post",
+    t.field('publish', {
+      type: 'Post',
       nullable: true,
       args: {
         id: idArg(),
@@ -108,15 +108,15 @@ export const schema = makeSchema({
   types: [Query, Mutation, Post, User],
   plugins: [nexusPrismaPlugin()],
   typegenAutoConfig: {
-    contextType: "Context.Context",
+    contextType: 'Context.Context',
     sources: [
       {
-        source: "@generated/photon",
-        alias: "photon",
+        source: '@generated/photon',
+        alias: 'photon',
       },
       {
-        source: require.resolve("./context.ts"),
-        alias: "Context",
+        source: require.resolve('./context'),
+        alias: 'Context',
       },
     ],
   },

--- a/typescript/graphql-auth/package.json
+++ b/typescript/graphql-auth/package.json
@@ -2,6 +2,7 @@
   "name": "graphql-auth",
   "license": "MIT",
   "scripts": {
+    "start": "node dist/server",
     "clean": "rm -rf dist",
     "build": "npm -s run clean && npm -s run generate && tsc",
     "generate": "npm -s run generate:prisma && npm -s run generate:nexus",

--- a/typescript/graphql-auth/src/schema.ts
+++ b/typescript/graphql-auth/src/schema.ts
@@ -1,6 +1,6 @@
-import { nexusPrismaPlugin } from "nexus-prisma"
-import { makeSchema } from "nexus"
-import * as types from "./types"
+import { nexusPrismaPlugin } from 'nexus-prisma'
+import { makeSchema } from 'nexus'
+import * as types from './types'
 
 export const schema = makeSchema({
   types,
@@ -8,14 +8,14 @@ export const schema = makeSchema({
   typegenAutoConfig: {
     sources: [
       {
-        source: "@generated/photon",
-        alias: "photon",
+        source: '@generated/photon',
+        alias: 'photon',
       },
       {
-        source: require.resolve("./context.ts"),
-        alias: "Context",
+        source: require.resolve('./context'),
+        alias: 'Context',
       },
     ],
-    contextType: "Context.Context",
+    contextType: 'Context.Context',
   },
 })

--- a/typescript/graphql/package.json
+++ b/typescript/graphql/package.json
@@ -1,6 +1,7 @@
 {
   "name": "typescript-graphql",
   "scripts": {
+    "start": "node dist/server",
     "clean": "rm -rf dist",
     "build": "npm -s run clean && npm -s run generate && tsc",
     "generate": "npm -s run generate:prisma && npm -s run generate:nexus",

--- a/typescript/graphql/src/schema.ts
+++ b/typescript/graphql/src/schema.ts
@@ -1,8 +1,8 @@
-import { nexusPrismaPlugin } from "nexus-prisma"
-import { idArg, makeSchema, objectType, stringArg } from "nexus"
+import { nexusPrismaPlugin } from 'nexus-prisma'
+import { idArg, makeSchema, objectType, stringArg } from 'nexus'
 
 const User = objectType({
-  name: "User",
+  name: 'User',
   definition(t) {
     t.model.id()
     t.model.name()
@@ -14,7 +14,7 @@ const User = objectType({
 })
 
 const Post = objectType({
-  name: "Post",
+  name: 'Post',
   definition(t) {
     t.model.id()
     t.model.createdAt()
@@ -27,14 +27,14 @@ const Post = objectType({
 })
 
 const Query = objectType({
-  name: "Query",
+  name: 'Query',
   definition(t) {
     t.crud.post({
-      alias: "post",
+      alias: 'post',
     })
 
-    t.list.field("feed", {
-      type: "Post",
+    t.list.field('feed', {
+      type: 'Post',
       resolve: (_parent, _args, ctx) => {
         return ctx.photon.posts.findMany({
           where: { published: true },
@@ -42,8 +42,8 @@ const Query = objectType({
       },
     })
 
-    t.list.field("filterPosts", {
-      type: "Post",
+    t.list.field('filterPosts', {
+      type: 'Post',
       args: {
         searchString: stringArg({ nullable: true }),
       },
@@ -62,13 +62,13 @@ const Query = objectType({
 })
 
 const Mutation = objectType({
-  name: "Mutation",
+  name: 'Mutation',
   definition(t) {
-    t.crud.createOneUser({ alias: "signupUser" })
+    t.crud.createOneUser({ alias: 'signupUser' })
     t.crud.deleteOnePost()
 
-    t.field("createDraft", {
-      type: "Post",
+    t.field('createDraft', {
+      type: 'Post',
       args: {
         title: stringArg({ nullable: false }),
         content: stringArg(),
@@ -88,8 +88,8 @@ const Mutation = objectType({
       },
     })
 
-    t.field("publish", {
-      type: "Post",
+    t.field('publish', {
+      type: 'Post',
       nullable: true,
       args: {
         id: idArg(),
@@ -108,15 +108,15 @@ export const schema = makeSchema({
   types: [Query, Mutation, Post, User],
   plugins: [nexusPrismaPlugin()],
   typegenAutoConfig: {
-    contextType: "Context.Context",
+    contextType: 'Context.Context',
     sources: [
       {
-        source: "@generated/photon",
-        alias: "photon",
+        source: '@generated/photon',
+        alias: 'photon',
       },
       {
-        source: require.resolve("./context.ts"),
-        alias: "Context",
+        source: require.resolve('./context'),
+        alias: 'Context',
       },
     ],
   },

